webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const path = require("path");
+const autoprefixer = require("autoprefixer");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+module.exports = [
+  {
+    entry: ["./src/index.scss", "./src/index.js"],
+    output: {
+      path: path.resolve(__dirname, "./dist"),
+      filename: "main.js"
+    },
+
+    module: {
+      rules: [
+        {
+          test: /\.scss$/,
+          use: ExtractTextPlugin.extract({
+            use: [
+              {
+                loader: "css-loader",
+                options: { sourceMap: true }
+              },
+              {
+                loader: "postcss-loader",
+                options: {
+                  sourceMap: true,
+                  plugins: [
+                    autoprefixer({
```

Comment:
why you create postcss.config.js and mention the same here? pick either postscss.config.js or this way